src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+@use '../_variables/variables' as *;
+
+.page {
+  font-family: $PrimaryFont;
+  color: $ColorMainAccent;
```

Comment:
The variable `$ColorMainAccent` should follow the naming convention for SCSS variables, which typically use lowercase letters and hyphens. Consider renaming it to `$color-main-accent`.